internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -259,50 +217,25 @@ func setCPUSLoadBalancing(c *oci.Container, enable bool, schedDomainDir string)
 		return fmt.Errorf("find container %s CPUs", c.ID())
 	}
 
-	cpus, err := cpuset.Parse(lspec.Resources.CPU.Cpus)
-	if err != nil {
-		return err
+	if node.CgroupIsV2() {
```

Comment:
Will this cause a container launch failure when the annotation is present on a system with cgroups v2? That would be a serious regression if that is the end results.

Reply:
Or maybe it is actually a good thing and we will fix it later when the v2 support lands.

Reply:
There is no support for this functionality in v2.

Reply:
unfortunately this needs to be the case. The cpuset controller for cgroup2 needs some modifications to work with kubernetes because of the unified hierarchy and new knobs that control this

Reply:
Yep, but maybe the error message should be a bit more explicit about what the issue is.

Reply:
updated, is that better @MarSik ?

Reply:
Much better, thanks.